ui/raidboss/data/triggers/suzaku-ex.js

```diff
@@ -38,7 +38,6 @@
     {
       id: 'SuzEx Phantom Flurry',
       regex: / 14:32DC:Suzaku starts using/,
-      regexFr: / 14:32DC:Suzaku prépare/,
```

Comment:
Did you mean to remove this?

Reply:
The initial EN trigger is working, so I removed the translation.